Source/Core/DolphinWX/LuaScriptFrame.cpp

```diff
@@ -0,0 +1,553 @@
+// Copyright 2017 Dolphin Emulator Project
+// Licensed under GPLv2+
+// Refer to the license.txt file included.
+
+#pragma once
+
+#include "LuaScripting.h"

```

Comment:
Include should be relative (`"Dolphin/LuaScripting.h"`). This should also be with the rest of the DolphinWX includes below.